Analysis/Tasks/PWGHF/HFLcCandidateSelector.cxx

```diff
@@ -190,17 +190,17 @@ struct HFLcCandidateSelector {
   template <typename T>
   bool selectionPIDTPC(const T& track, int nPDG, int nSigmaCut)
   {
-    double nSigma = 100.0; //arbitarily large value
+    double nSigma = 1.0; //arbitarily large value
```

Comment:
If I understand it right, you are setting the PID to 1 meaning to a small value so the candidates are always selected. Am I correct?

Reply:
If what i got is right, you should adjust the comment to the line.

Reply:
yes exactly. The PID should be tested, so for the moment i'm not using it putting a small value. But if you want to enable the PID a large value must be setted.